apps/ledger-live-desktop/src/renderer/hooks/useNetworkStatus.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+
+export enum NetworkStatus {
+  OFFLINE = "offline",
+  ONLINE = "online",
+}
+export const useNetworkStatus = () => {
+  const [networkStatus, setNetworkStatus] = useState<NetworkStatus>(
+    navigator.onLine ? NetworkStatus.ONLINE : NetworkStatus.OFFLINE,
```

Comment:
Can you add a small comment saying you first check for the initial network connection thanks to the (window) `navigator` HTML5 API that we can access from Electron ?

The `navigator` variable could be confusing at first (maybe compared to the `window` one that is more well-known)